.gitignore

```diff
@@ -3,3 +3,6 @@
 .build
 *.swp
 release-tools
+vendor
+.idea
+travis.yml
```

Comment:
travis.yml shouldn't be ignored. no?

Reply:
I'm not sure, it is ignored in the sidecar, so I added the same ignore here as well.